src/azure-cli/azure/cli/command_modules/storage/_transformers.py

```diff
@@ -167,3 +169,54 @@ def transform_fs_list_public_access_output(result):
 
 def transform_metadata(result):
     return result.metadata
+
+
+# ------------------Track2 Support-----------------------
+def _transform_page_ranges(page_ranges):
+    # in track 2 sdk, page ranges result is tuple(list(dict(str, str), list(dict(str, str))
+    if page_ranges and len(page_ranges) == 2:
+        result = page_ranges[0] if page_ranges[0] else [{}]
+        result[0]['isCleared'] = bool(page_ranges[1])
+        return result
+    return None
+
+
+def transform_blob_json_output(result):
+    result = todict(result)
+    new_result = {
+        "content": "",
+        "deleted": result.pop('deleted', None),
+        "metadata": result.pop('metadata', None),
+        "name": result.pop('name', None),
+        "properties": {
+            "appendBlobCommittedBlockCount": result.pop('appendBlobCommittedBlockCount', None),
+            "blobTier": result.pop('blobTier', None),
+            "blobTierChangeTime": result.pop('blobTierChangeTime', None),
+            "blobTierInferred": result.pop('blobTierInferred', None),
+            "blobType": result.pop('blobType', None),
+            "contentLength": result.pop('size', None),
+            "contentRange": result.pop('contentRange', None),
+            "contentSettings": {
+                "cacheControl": result['contentSettings']['cacheControl'],
+                "contentDisposition": result['contentSettings']['contentDisposition'],
+                "contentEncoding": result['contentSettings']['contentEncoding'],
+                "contentLanguage": result['contentSettings']['contentLanguage'],
+                "contentMd5": _encode_bytes(result['contentSettings']['contentMd5']),
+                "contentType": result['contentSettings']['contentType']
+            },
```

Comment:
May I ask does the `contentSettings` must exist in the result? Because these are not required fields, It would cause `KeyError` if it didn't exist

Reply:
They should always in response although sometimes the values are None.